.github/workflows/main.yml

```diff
@@ -98,8 +98,6 @@ jobs:
             osName: windows
           - os: macOS-latest
             osName: mac
```

Comment:
I think we need to also explicitly specify `deploy: false` on windows and mac os.